src/ui/common/src/components/integrations/integrationObjectList.tsx

```diff
@@ -54,6 +54,13 @@ const IntegrationObjectList: React.FC<Props> = ({ user, integration }) => {
     );
   }, [selectedObject]);
 
+  if (
```

Comment:
Sounds like a better idea, I will make edition based on that.

Reply:
Not sure if this is what you want, but if you want to return nothing, you can just return ```null``` here.

Reply:
I meant rather than returning `null` / nothing / blank / empty, we should put an infobox saying "We do not support listing the objects in this integration" similar to S3.

Reply:
@eunice-chan yeah, i understood you there, just wanted to point out that instead of fragments we can have null if there's nothing you wish to show.

Would you or @Fanjia-Yan mind making a linear task to track the work of setting up an alert box here? Please post a link here once done adding to backlog.

Reply:
@agiron123 Ooh okay, I thought there was a misunderstanding. Yes, thank you for the tip!

Created the linear task here: https://linear.app/aqueducthq/issue/ENG-1694/lambda-and-kubernetes-integration-object-list-add-alert @Fanjia-Yan assigned you to work on it for now.